source/funwithflags/gateways/context.py

```diff
@@ -0,0 +1,16 @@
+"""Definition of processing context."""
+from dataclasses import dataclass
+
+from .db_gateway import make_postgres_gateway, PostgresGateway
+
+
+@dataclass
+class Context:
+    postgres_gateway: PostgresGateway
+
+
+def make_context(postgres_gateway=None) -> Context:
```

Comment:
This function isn't really necessary. Move the logic into constructor of context.

Reply:
Done.